### PR TITLE
Retest/stable

### PR DESCRIPTION
**Pull request by timofeevmd**

### CODE REVIEW

File: src/test/java/configs/tests/PalauPrecondition.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+public class PalauPrecondition  {
+
+    public static ArrayList<String> assetDefinitionIds;
+
+    public static ArrayList<String> domainIds;
+
+    public static ArrayList<String> accountIds;
+
+    private final String path = "/Users/michaeltimofeev/Documents/projects/iroha2-perf/src/test/resources/iroha2_config/stable/5d44d59/";
```
Comment (adaagava) [inline src/test/java/configs/tests/PalauPrecondition.java:19]: Please use relative path or env.variable here

File: src/test/java/healper/impl/SmartContractImpl.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    @SneakyThrows
+    private SignedTransaction registerRegisterBondTrigger(String registerBondId, String registerBondWasm) {
+        final var triggerId = new TriggerId(null, ExtensionsKt.asName(registerBondId));
+
+        final var filter = buildFilterByTriggerMetadataInserted(triggerId);
+
+        final var wasm = Objects
+                .requireNonNull(getClass().getClassLoader().getResource(registerBondWasm))
+                .openConnection()
```
Comment (adaagava) [inline src/test/java/healper/impl/SmartContractImpl.java:110]: Shouldn't the connection be closed after?

File: src/test/java/scenarios/DefinitionId.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package scenarios;
+
+public class DefinitionId {
+    /*public static ScenarioBuilder definitionId = scenario("register definition id")
```
Comment (adaagava) [inline src/test/java/scenarios/DefinitionId.java:4]: Should it be uncommented?

File: src/test/java/healper/models/Domain.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package healper.models;
```
Comment (adaagava) [inline src/test/java/healper/models/Domain.java:1]: healper - nice name!

File: src/test/java/requests/Triggers.java [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+    public static ChainBuilder registerBondAssetTrigger =
+            // it must use onlyOne() controller
+            exec(http("register bond_asset trigger")
+                    .post(session -> {
+                                return "/peer-0/" + Constants.URL_TRANSACTION;
+                            }
+                    )
+                    .body(ByteArrayBody(session -> {
+                                        return SignedTransaction.Companion.encode(smartContract.deployRegisterRegisterBondTrigger("019153fc-8d19-7faa-854a-57657ba0ed20"));
```
Comment (adaagava) [inline src/test/java/requests/Triggers.java:39]: What's it? Should it be a predefined constant?

**Review by adaagava — CHANGES_REQUESTED** [review]

File: src/test/java/healper/impl/SmartContractImpl.java (hunk 2 above)
Comment (timofeevmd) [reply src/test/java/healper/impl/SmartContractImpl.java]: it was closed automatically

File: src/test/java/configs/tests/PalauPrecondition.java (hunk 1 above)
Comment (timofeevmd) [reply src/test/java/configs/tests/PalauPrecondition.java]: done

File: src/test/java/requests/Triggers.java (hunk 5 above)
Comment (timofeevmd) [reply src/test/java/requests/Triggers.java]: done

File: src/test/java/healper/models/Domain.java (hunk 4 above)
Comment (timofeevmd) [reply src/test/java/healper/models/Domain.java]: renamed

File: performance-generator/src/test/java/functionalTest/SendQuery.java [hunk 6]
```diff
@@ -34,9 +27,9 @@ public void findAllAssets() {
 
     private Iroha2AsyncClient builderAsyncClient(String targetPeer) throws MalformedURLException {
         List<IrohaUrls> irohaUrls = new ArrayList<>();
-        irohaUrls.add(new IrohaUrls("https://iroha2.test.tachi.soramitsu.co.jp", targetPeer));
+        /*irohaUrls.add(new IrohaUrls("https://iroha2.test.tachi.soramitsu.co.jp", targetPeer));
```
Comment (adaagava) [inline performance-generator/src/test/java/functionalTest/SendQuery.java:30]: Let's remove it if not needed?

File: performance-generator/pom.xml [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+    <dependencies>
+        <dependency>
+            <groupId>io.gatling</groupId>
+            <artifactId>gatling-core-java</artifactId>
+            <version>${gatling.version}</version>
+        </dependency>
+        <!--dependency>
+            <groupId>io.gatling</groupId>
+            <artifactId>gatling-http</artifactId>
```
Comment (adaagava) [inline performance-generator/pom.xml:33]: please remove is not needed

File: performance-generator/pom.xml [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>16</source>
+                    <target>16</target>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
```
Comment (adaagava) [inline performance-generator/pom.xml:64]: Don't we have Lompoc above already?

**Review by adaagava — CHANGES_REQUESTED** [review]

File: performance-generator/pom.xml (hunk 7 above)
Comment (timofeevmd) [reply performance-generator/pom.xml:33]: deleted

File: performance-generator/pom.xml (hunk 8 above)
Comment (timofeevmd) [reply performance-generator/pom.xml:64]: deleted

File: performance-generator/src/test/java/functionalTest/SendQuery.java (hunk 6 above)
Comment (timofeevmd) [reply performance-generator/src/test/java/functionalTest/SendQuery.java:30]: was fixed

**Review by adaagava — APPROVED** [review]

File: performance-generator/src/test/resources/.DS_Store
Comment (adaagava) [inline performance-generator/src/test/resources/.DS_Store:1]: please remove and add to gitignore

File: performance-generator/src/test/resources/iroha2_config/stable/5d44d59/.DS_Store
Comment (adaagava) [inline performance-generator/src/test/resources/iroha2_config/stable/5d44d59/.DS_Store:1]: please remove and add to gitignore

File: performance-generator/target/classes/configs/tests/PalauProperties$TriggerProperties.class
Comment (adaagava) [inline performance-generator/target/classes/configs/tests/PalauProperties$TriggerProperties.class:1]: Looks like all target holder should be removed

File: util/target/util-1.0.jar
Comment (adaagava) [inline util/target/util-1.0.jar:1]: Looks like all target folder should be removed

**Review by adaagava — CHANGES_REQUESTED** [review]

**Pull request closed without merging**